FWCore/Services/test/BuildFile.xml

```diff
@@ -9,11 +9,12 @@
   <use name="FWCore/Framework"/>
 </library>
 
+<test name="TestFWCoreServicesDriver_sitelocalconfig" command="test_sitelocalconfig.sh"/>
+<test name="TestFWCoreServicesDriver_zombiekiller"    command="test_zombiekiller.sh"/>
+<test name="TestFWCoreServicesDriver_cpu"             command="test_cpu.sh"/>
+
 <ifrelease name="!_ASAN_">
-  <test name="TestFWCoreServicesDriver_sitelocalconfig" command="test_sitelocalconfig.sh"/>
   <test name="TestFWCoreServicesDriver_resource"        command="test_resource.sh"/>
-  <test name="TestFWCoreServicesDriver_zombiekiller"    command="test_zombiekiller.sh"/>
```

Comment:
`TestFWCoreServicesDriver_resource` still fails with error
```
Exceeded maximum allowed VSize of 1 GB (VSize is 20972.2)
```
this is expected. May be we can update the test and pass the expected memory usage as input parameter?